Add CMS Guides landing page (#1975)
withastro/docs: src/pages/en/guides/cms.md

---
title: Use a CMS with Astro
description: How to use a CMS to add content to Astro
layout: ~/layouts/MainLayout.astro
setup: 
i18nReady: true
---

> **Ready to connect a Headless CMS to your Astro project?** Follow one of our guides to integrate a CMS or scroll down for general guidance about deploying an Astro site.

Reviewer: ```suggestion
**Ready to connect a Headless CMS to your Astro project?** Follow one of our guides to integrate a CMS.
```

Author: 🤦

Note that many of these pages are **stubs**: they're collections of resources waiting for your contribution!

## Why use a CMS?

A Content Managament System lets you write content and manage assets outside of your Astro project.

This unlocks new features for working with content. Most CMSes give you a visual content editor, the ability to specify standard types of content, and a way to collaborate with others.

> It's up to you to determine what content on your site is managed by the CMS, and where on your site you want to display it.

Reviewer: ```suggestion

It's up to you to determine what content on your site is managed by the CMS, and where on your site you want to display it.
```
Did you want this to be its own paragraph?

> You might use a CMS to build a blog for someone who doesn't know Markdown - they can write posts using a CMS's rich text editor instead. If you're building an ecommerce shop, you might use a CMS to publish and edit shop items. You can specify the fields that all shop items must have, and your Astro project can then fetch and display them.

Reviewer: ```suggestion
You might use a CMS to build a blog for someone who doesn't know Markdown — they can write posts using a CMS's rich text editor instead. If you're building an ecommerce shop, you might use a CMS to organize and edit shop items. You can configure your CMS to hold certain pieces of information about each shop item, even making them required to avoid incomplete listings, and your Astro project can then fetch and display them.
```
"publish" shop items felt a little weird -- jargony? Maybe something more related to "organizing" or "maintaining"?

Also tried to think more user-y re: shop items?

## Which CMSes work well with Astro?

> Because Astro takes care of the _presentation_ of your content, you'll want to choose a _headless_ CMS, like those in the list above. This means that the CMS helps you write your content, but doesn't generate a site that displays it. Instead, you fetch the content data and use in your Astro project. 

Reviewer: ```suggestion
Because Astro takes care of the _presentation_ of your content, you'll want to choose a _headless_ CMS, like those in the list below. This means that the CMS helps you write your content, but doesn't generate a site that displays it. Instead, you fetch the content data hosted there and use in your Astro project. 
```
I mean, depends on what the page looks like! I figured, like deploy, the content would come BEFORE all the icons.  😅

Author: On deploy the icons come first! I don't think we want them to scroll through the content first

> Some CMSes, like Storyblok, provide an Astro [integration](https://docs.astro.build/en/guides/integrations-guide/) that helps fetch the content specifically for an Astro site. Others provide a JavaScript SDK, a library that you install and use to fetch your remote content.
> 
> ## Can I use Astro without a CMS?
> 
> Yes! Astro provides built-in ways to [author content](https://docs.astro.build/en/guides/content/), including support for Markdown pages.

Reviewer: ```suggestion
Some CMSes, like Storyblok, provide an Astro [integration](/en/guides/integrations-guide/) that helps fetch the content specifically for an Astro site. Others provide a JavaScript SDK, a library that you install and use to fetch your remote content.

## Can I use Astro without a CMS?

Yes! Astro provides built-in ways to [author content](/en/guides/content/), including support for Markdown pages.
```

---
title: Use a CMS with Astro
description: How to use a CMS to add content to Astro
layout: ~/layouts/MainLayout.astro
setup: |
  import CMSGuidesNav from '~/components/CMSGuidesNav.astro';
i18nReady: true
---

**Ready to connect a Headless CMS to your Astro project?** Follow one of our guides to integrate a CMS.

## CMS Guides

<CMSGuidesNav />

Note that many of these pages are **stubs**: they're collections of resources waiting for your contribution!

## Why use a CMS?

> A Content Managament System lets you write content and manage assets outside of your Astro project.
> 
> This unlocks new features for working with content. Most CMSes give you a visual content editor, the ability to specify standard types of content, and a way to collaborate with others. It's up to you to determine what content on your site is managed by the CMS, and where on your site you want to display it.
> 
> You might use a CMS to build a blog for someone who doesn't know Markdown — they can write posts using a CMS's rich text editor instead. If you're building an ecommerce shop, you might use a CMS to maintain shop items. You can configure your CMS to hold certain pieces of information about each shop item, even making them required to avoid incomplete listings, and your Astro project can then fetch and display them.

Reviewer: ```suggestion
A Content Managament System lets you write content and manage assets outside of your Astro project.

This unlocks new features for working with content. Most CMSes give you a visual content editor, the ability to specify standard types of content, and a way to collaborate with others. 

A CMS can be useful for content that follows a particular structure, often giving you a dashboard-like experience and WYSIWYG editing tools. You can use a CMS to write blog posts using a CMS's rich text editor instead of Markdown files. You can also use a CMS to maintain product listings for an eCommerce shop, even making certain fields required to avoid incomplete listings. 

Your Astro project can then fetch and display your content from the CMS, wherever on your site you want to display it.
```

Thoughts?

---
title: Use a CMS with Astro
description: How to use a CMS to add content to Astro
layout: ~/layouts/MainLayout.astro
setup: |
  import CMSGuidesNav from '~/components/CMSGuidesNav.astro';
i18nReady: true
---

**Ready to connect a Headless CMS to your Astro project?** Follow one of our guides to integrate a CMS.

## CMS Guides

<CMSGuidesNav />

Note that many of these pages are **stubs**: they're collections of resources waiting for your contribution!

## Why use a CMS?

A Content Management System lets you write content and manage assets outside of your Astro project.

This unlocks new features for working with content. Most CMSes give you a visual content editor, the ability to specify standard types of content, and a way to collaborate with others. 

A CMS can be useful for content that follows a particular structure, often giving you a dashboard-like experience and WYSIWYG editing tools. You might use a CMS to write blog posts using a CMS's rich text editor instead of Markdown files. Or you might use a CMS to maintain product listings for an eCommerce shop, making certain fields required to avoid incomplete listings. 

> Your Astro project can then fetch your content from your CMS and display it, wherever you want on your site.

Reviewer: ```suggestion
Your Astro project can then fetch your content from your CMS and display it, wherever and however you want on your site.
```
Do we like this? @Jutanium

Author: It isn't clear what the different "however" possibilities are within "displaying" it - how about:

> Your Astro project can then fetch your content from your CMS and use it, wherever and however you want on your site.
